tests/cloudformation/checks/resource/aws/example_AmazonMQBrokerPublicAccess/AmazonMQBrokerPublicAccess-FAILED-1.yaml

```diff
@@ -0,0 +1,33 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Create a single-instance Amazon MQ for RabbitMQ broker without public accessibility
+
+Parameters:
+  AmazonMqPassword:
+    Type: String
+    NoEcho: true
+  AmazonMqUsername:
+    Type: String
+
+Resources:
+  PublicBroker0:
+    Type: 'AWS::AmazonMQ::Broker'
+    Properties:
+      AutoMinorVersionUpgrade: false
+      BrokerName: MyComplexRabbitBroker-Yes
+      DeploymentMode: SINGLE_INSTANCE
+      EngineType: RABBITMQ
+      EngineVersion: 3.8.6
+      HostInstanceType: mq.t3.micro
+      Logs:
+        General: true
+      MaintenanceWindowStartTime:
+        DayOfWeek: Monday
+        TimeOfDay: '22:45'
+        TimeZone: America/Los_Angeles
+      PubliclyAccessible: Yes
```

Comment:
just out of curiosity, is Yes and No valid values for a boolean value in CloudFormation?

Reply:
@gruebel yes/no/true/false/on/off all are valid boolean values in CloudFormation

Reply:
@gruebel yes/no/true/false/on/off are all these valid boolean values in CloudFormation. I have tested these.